app/admin/models/Mealtimes_model.php

```diff
@@ -50,13 +50,13 @@ public function isAvailable($datetime = null)
         }
 
         return $datetime->between(
-            Carbon::createFromTimeString($this->start_time),
-            Carbon::createFromTimeString($this->end_time)
+            Carbon::createFromFormat('Y-m-d H:i:s', $datetime->format('Y-m-d ').$this->start_time),
+            Carbon::createFromFormat('Y-m-d H:i:s', $datetime->format('Y-m-d ').$this->end_time)
```

Comment:
How about setting the time on `$datetime` like:
```php
return $datetime->between(
    $datetime->copy()->setTimeFromTimeString($this->start_time),
    $datetime->copy()->setTimeFromTimeString($this->end_time)
);
```
Not tested, BTW.

Reply:
Yep that looks to work. I've updated the PR.